### PR TITLE
@luc/fix root collection

### PR DESCRIPTION
**Pull request by LucRoy**
#### What is the purpose of this change?

- Fixed copying files to estuary root
- Fixed listing files without modTime

### CODE REVIEW

File: backend/estuary/estuary.go [hunk 1]
```diff
@@ -436,12 +436,16 @@ func (f *Fs) List(ctx context.Context, dir string) (entries fs.DirEntries, err e
 			if err != nil {
 				return nil, err
 			}
-			o, err := f.newObjectWithInfo(ctx, path.Join(dir, item.Name), parseTimeString(pin.Meta["modTime"].(string)), &content{ID: item.ContentID, Cid: item.Cid, Size: item.Size})
+			modTime := pin.Meta["modTime"]
+			if modTime == nil {
+				// Only object created by rclone will have modTime
+				modTime = "0"
```
Comment (anjor) [inline backend/estuary/estuary.go]: can we change this to the current time or something? Let's default to something better than 0.

File: backend/estuary/estuary.go [hunk 2]
```diff
@@ -436,12 +436,16 @@ func (f *Fs) List(ctx context.Context, dir string) (entries fs.DirEntries, err e
 			if err != nil {
 				return nil, err
 			}
-			o, err := f.newObjectWithInfo(ctx, path.Join(dir, item.Name), parseTimeString(pin.Meta["modTime"].(string)), &content{ID: item.ContentID, Cid: item.Cid, Size: item.Size})
+			modTime := pin.Meta["modTime"]
+			if modTime == nil {
+				// Only object created by rclone will have modTime
```
Comment (anjor) [inline backend/estuary/estuary.go]: Let's remove this comment actually because this can change in other codebases that we don't have control over.

**Comment by anjor** [comment]
a couple of minor comments, but otherwise lgtm.

File: backend/estuary/estuary.go (hunk 2 above)
Comment (LucRoy) [reply backend/estuary/estuary.go]: removed

File: backend/estuary/estuary.go (hunk 1 above)
Comment (LucRoy) [reply backend/estuary/estuary.go]: changed it to use current time

**Pull request merged**